_posts/databases/postgresql/timescaledb/2000-01-01-downsampling.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Downsampling
+nav: Downsampling
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions downsampling
+---
+
+Downsampling data allows to reduce the amount of data by simplifying the old values.
```

Comment:
Simplification suggestion:

```suggestion
The downsampling reduces the amount of data by simplifying the old values.
```

I don't like the word "simplifying". I don't understand what it means